tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_ha_metrics.py

```diff
@@ -0,0 +1,885 @@
+                f"{snaps_synced_on_node1} and {snaps_synced_on_node2}"
+            )
+        else:
+            log.error(
+                f"Snapshot created count : {snap_count} doesn't exist in Metrics : "
+                f"{snaps_synced_on_node1} and {snaps_synced_on_node2}"
+            )
+            raise CommandFailed("Snapshot created count is mismatching in Metrics")
+
+        snaps_deleted_on_node1 = counter_on_node1.get("snaps_deleted")
+        snaps_deleted_on_node2 = counter_on_node2.get("snaps_deleted")
+        log.info(
+            f"Snaps Deleted: {snaps_deleted_on_node1} and {snaps_deleted_on_node2}"
+        )
+
+        snaps_renamed_on_node1 = counter_on_node1.get("snaps_renamed")
+        snaps_renamed_on_node2 = counter_on_node2.get("snaps_renamed")
+        log.info(
+            f"Snaps Renamed: {snaps_renamed_on_node1} and {snaps_renamed_on_node2}"
+        )
+
+        sync_failures_on_node1 = counter_on_node1.get("sync_failures")
+        sync_failures_on_node2 = counter_on_node2.get("sync_failures")
+        log.info(
+            f"Sync Failures: {sync_failures_on_node1} and {sync_failures_on_node2}"
+        )
+
+        avg_sync_count_on_node1 = counter_on_node1.get("avg_sync_time", {}).get(
+            "avgcount"
+        )
+        avg_sync_count_on_node2 = counter_on_node2.get("avg_sync_time", {}).get(
+            "avgcount"
+        )
+        log.info(
+            f"Average Sync Count: {avg_sync_count_on_node1} and {avg_sync_count_on_node2}"
+        )
+
+        sum_on_node1 = counter_on_node1.get("avg_sync_time", {}).get("sum")
+        sum_on_node2 = counter_on_node2.get("avg_sync_time", {}).get("sum")
+        log.info(f"Average Sync Sum: {sum_on_node1} and {sum_on_node2}")
+
+        avg_time_on_node1 = counter_on_node1.get("avg_sync_time", {}).get("avgtime")
+        avg_time_on_node2 = counter_on_node2.get("avg_sync_time", {}).get("avgtime")
+        log.info(f"Average Sync Time: {avg_time_on_node1} and {avg_time_on_node2}")
+
+        sync_bytes_on_node1 = counter_on_node1.get("sync_bytes")
+        sync_bytes_on_node2 = counter_on_node2.get("sync_bytes")
+        log.info(f"Sync Bytes: {sync_bytes_on_node1} and {sync_bytes_on_node2}")
+
+        log.info(
+            "Validate if all the details of Cephfs Mirrors are displayed in the counters"
+        )
+        report_on_node1 = cephfs_mirror_node1_data["cephfs_mirror"]
+        report_on_node2 = cephfs_mirror_node2_data["cephfs_mirror"]
+        for report_name, report_data in [
+            ("report1", report_on_node1),
+            ("report2", report_on_node2),
+        ]:
+            for value in report_data:
+                counters = value.get("counters", {})
+                mirrored_fs = counters.get("mirrored_filesystems", "")
+                mirror_enable_failure = counters.get("mirror_enable_failures", "")
+
+                if initial_fs_count == mirrored_fs:
```

Comment:
We are validating only fs_count value here, are there any other details to be validated from report generated at line283?
Am asking this question due to log.info "Validate if all the details of Cephfs Mirrors are displayed in the counters"

Reply:
We have 3 different sections in counters : 
1. cephfs_mirror_mirrored_filesystems
2. cephfs_mirror_peers
3. cephfs_mirror
Each sections will have it's own counters : 
log.info "Validate if all the details of Cephfs Mirrors are displayed in the counters" - This will validate only counters part of "cephfs_mirror"
Under this - we only have counters for "mirrored_filesystems" and "mirror_enable_failures"